substrate/client/transaction-pool/src/fork_aware_txpool/view.rs

```diff
@@ -184,8 +189,12 @@ where
 		xt: ExtrinsicFor<ChainApi>,
 	) -> Result<ValidatedPoolSubmitOutcome<ChainApi>, ChainApi::Error> {
 		let (hash, length) = self.pool.validated_pool().api().hash_and_length(&xt);
-		log::trace!(target: LOG_TARGET, "[{:?}] view::submit_local at:{}", hash, self.at.hash);
-
+		tracing::trace!(
+			target: LOG_TARGET,
+			?hash,
```

Comment:
```suggestion
			?tx_hash,
```

rename `hash` -> `tx_hash`